packages/jest-schemas/src/raw-types.ts

```diff
@@ -226,129 +226,145 @@ const RawHasteConfig = Type.Partial(
 );
 
 export const RawInitialOptions = Type.Partial(
-  Type.Object({
```

Comment:
ignore whitespace (https://github.com/jestjs/jest/pull/14792/files?w=1) - I just added some meta properties at the end